src/routes/(previews)/Separator/example.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+	import { Separator } from '$lib';
+</script>
```

Comment:
Ideally we export propsObj, just as the other examples. Ping me if you need any more info on that 😄

Reply:
Disregard that! I saw the example now 😄